examples/pom.xml

```diff
@@ -64,6 +64,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
+        <version>${maven-checkstyle-plugin.version}</version>
```

Comment:
Shouldn't this be inherited from parent?

Reply:
Yes in principle I put it just because the command `mvn versions:display-plugin-updates` showed a warning of type version not updated but curiously inferred the right version. I can let you as you prefer, remove it from everywhere or make it explicit everywhere. What should I do ?

Reply:
So weird. That's the whole point of `pluginManagement` isn't it?

Reply:
Agree @kennknowles but there's `<reporting/>` and `<pluginManagement/>`. It should go there.